src/index.ts

```diff
@@ -393,7 +393,7 @@ export function cast(field: Field, value: string | null): any {
     case 'UINT24':
     case 'UINT32':
     case 'YEAR':
-      return parseInt(value, 10)
+      return value ? parseInt(value, 10) : value
```

Comment:
imo, arguably here, I don't know what is right, but if we don't do this, it'll return NaN, which sorta feels more right and is frankly a bug somewhere.

That case is, "we got a number field, but no data." which... should be entirely wrong in every case.